client/state/sharing/publicize/test/reducer.js

```diff
@@ -48,8 +50,14 @@ describe( '#fetchingConnections()', () => {
 	} );
 
 	describe( 'persistence', () => {
+		before( () => {
+			sinon.stub( console, 'warn' );
+		} );
+		after( () => {
+			console.warn.restore();
+		} );
 		it( 'should load valid persisted data', () => {
-			const persistedState = Object.freeze( {
+			const persistedState = deepFreeze( {
```

Comment:
A minor point, while I think it's usually fine to sneak in small maintenance changes, it'd probably be best to do so as separate commits to keep history meaningful, and to enable a reviewer to review the commits in isolation.